src/net/rpc.rs

```diff
@@ -475,7 +476,7 @@ impl RPCPoxInfoData {
         let cur_cycle_pox_active = sortdb.is_pox_active(burnchain, &burnchain_tip)?;
 
         Ok(RPCPoxInfoData {
-            contract_id: boot_code_id(cur_cycle_pox_contract, chainstate.mainnet).to_string(),
+            contract_id: boot_code_id(cur_block_pox_contract, chainstate.mainnet).to_string(),
```

Comment:
Should there be a new entry in the RPC info for the current cycle's PoX contract?

Reply:
I'm not sure - No mention was made in the requesting ticket for a reward-based active contract field. https://github.com/stacks-network/stacks-blockchain/issues/3422 

maybe @janniks has some input.